src/operator/loss_binary_op.cc

```diff
@@ -9,7 +9,23 @@ namespace mxnet {
 namespace op {
 
 NNVM_REGISTER_OP(softmax_cross_entropy)
-.MXNET_DESCRIBE("Calculate cross_entropy(data, one_hot(label))")
+.describe(R"code(Calculate cross entropy of softmax output and one-hot label.
+
+- This operator is finished in two steps:
```

Comment:
What precisely is meant by "is finished"? Don't you mean "This operator applies computes the cross-entropy in two steps:"